packages/mermaid/src/mermaid.ts

```diff
@@ -3,64 +3,41 @@
  * functionality and to render the diagrams to svg code!
  */
 import dedent from 'ts-dedent';
-
 import { MermaidConfig } from './config.type';
 import { log } from './logger';
 import utils from './utils';
-import { mermaidAPI } from './mermaidAPI';
+import { mermaidAPI, ParseOptions, RenderResult } from './mermaidAPI';
 import { registerLazyLoadedDiagrams } from './diagram-api/detectType';
 import type { ParseErrorFunction } from './Diagram';
 import { isDetailedError } from './utils';
 import type { DetailedError } from './utils';
 import { registerDiagram } from './diagram-api/diagramAPI';
 import { ExternalDiagramDefinition } from './diagram-api/types';
 
-export type { MermaidConfig, DetailedError, ExternalDiagramDefinition, ParseErrorFunction };
+export type {
+  MermaidConfig,
+  DetailedError,
+  ExternalDiagramDefinition,
+  ParseErrorFunction,
+  RenderResult,
+  ParseOptions,
+};
 
 /**
- * ## init
+ * The options used when running mermaid.
  *
- * Function that goes through the document to find the chart definitions in there and render them.
- *
- * The function tags the processed attributes with the attribute data-processed and ignores found
- * elements with the attribute already set. This way the init function can be triggered several
- * times.
- *
- * ```mermaid
- * graph LR;
- *  a(Find elements)-->b{Processed}
- *  b-->|Yes|c(Leave element)
- *  b-->|No |d(Transform)
- * ```
- *
- * Renders the mermaid diagrams
- *
- * @param config - **Deprecated**, please set configuration in {@link initialize}.
- * @param nodes - **Default**: `.mermaid`. One of the following:
- * - A DOM Node
- * - An array of DOM nodes (as would come from a jQuery selector)
- * - A W3C selector, a la `.mermaid`
- * @param callback - Called once for each rendered diagram's id.
+ * @param querySelector - The query selector to use when finding elements to render. Default: .mermaid
+ * @param nodes - The nodes to render. If this is set, querySelector will be ignored.
+ * @param postRenderCallback - A callback to call after each diagram is rendered.
+ * @param suppressErrors - If true, errors will be logged to the console, but not thrown. Default: false
```

Comment:
I'm pretty sure that this should instead be documented as:

```ts
export interface RunOptions {
  /**
   * The query selector to use when finding elements to render. Default: `".mermaid"`.
   */ 
  querySelector?: string;
  /**
   * The nodes to render. If this is set, `querySelector` will be ignored.
   */
  nodes?: ArrayLike<HTMLElement>;
  /**
   * A callback to call after each diagram is rendered.
   */
  postRenderCallback?: (id: string) => unknown;
  /**
   * If `true`, errors will be logged to the console, but not thrown. Default: `false`
   */
  suppressErrors?: boolean;
}
```

Reply:
Done 38b2cbc3d195c14d0df780881022c05c184d4531